### ISSUE TITLE
'number-no-trailing-zeros' bug when using rgba($variable, $value) after 0, if turned on

### ISSUE THREAD

**Issue by Ne-Ne** [stylelint/stylelint#2012]
> Describe the issue. Is it a bug or a feature request (new rule, new option, etc.)?

As written in the description, number-no-trailing-zeros when set to true, using rgba in Sass (SCSS).

> Which rule, if any, is this issue related to?

number-no-trailing-zeros

> What CSS is needed to reproduce this issue?

e.g.

```scss
$black = rgb(0, 0, 0);

.class-name {
  box-shadow: inset 0 1px 5px 0 rgba($black, 0.10);
}
```

> What stylelint configuration is needed to reproduce this issue?

```json
{
  "rules": {
    "number-no-trailing-zeros": true
  }
}
```

> Which version of stylelint are you using?

```json
"gulp-stylelint": "3.4.0"
```

> How are you running stylelint: CLI, PostCSS plugin, Node API?

Gulp ^^, using gulp-sass.

> Does your issue relate to non-standard syntax (e.g. SCSS, nesting, etc.)?

Relates to box-shadow when a property is set after 0.

> What did you expect to happen?

No warnings to be flagged.

> What actually happened (e.g. what warnings or errors you are getting)?

e.g. "The following warnings were flagged:"

```shell
  58:53  ×  Unexpected trailing zero(s)   number-no-trailing-zeros
  145:51  ×  Unexpected trailing zero(s)   number-no-trailing-zeros
```

Many thanks.

**Comment by evilebottnawi** [comment]
@jeddy3 @davidtheclark i think we should add option `{ "ignore": ["inside-function"] }` as in https://github.com/stylelint/stylelint/issues/2009

**Comment by jeddy3** [comment]
I think the use case here is different to #2009.

In this case, I'm struggling to understand why you wouldn't just drop the trailing zero. That would make numbers consistent everywhere e.g.

```css
.class-name {
  box-shadow: inset 0 1px 5px 0 rgba($black, 0.1);
}
```

**Comment by davidtheclark** [comment]
Yeo ^^ Closing this because the rule is doing exactly what it's supposed to to.
